changelog.d/20240406_123538_athornton_DM_43720.md

```diff
@@ -0,0 +1,6 @@
+<!-- Delete the sections that don't apply -->
+
+### Backwards-incompatible changes
+
+Go to Jupyterhub 4.1.5, which tightens XSRF handling again.
```

Comment:
Needs to be a list. I think unless this breaks mobu we should mark this as a bug-fix change instead of backwards-incompatible, since at present I don't think we have any reason to believe upgrading the hub (as opposed to the lab) will break anything.